### PR TITLE
created: modal component

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Modal component 
I have added the modal component in component library.  So ignore other files. Focus only on the modal.html , modal.js, modal.js (in modal folder  ) ,  (in components) .

What not to check
Responsiveness
image

### CODE REVIEW

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    width: 30%;
+    border-radius: 5px;
+    position: relative;
+    padding: 4em;
+}
+
+#close {
+    display: inline-block;
+    font-size: 1.5rem;
+    padding: 0.5em;
```
Comment (developer-anand) [inline components/modal/modal.css:69]: Just a suggestion be consistent with styling units.

File: components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+}
+
+#close:hover {
+    cursor: pointer;
+    color: var(--button-secondary-color);
+    background-color: var(--danger-color);
+}
+
+#close-button {
+    margin: 0rem 2.5rem 1rem 0rem;
```
Comment (developer-anand) [inline components/modal/modal.css:83]: you can use px here.

**Review by Rockyraven — APPROVED** [review]
code is clean and easy to readable 
You are good to go

File: components/modal/modal.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const openModal = document.querySelector('#open-modal');
+const modal = document.querySelector('#modal');
+closeBtnTop = document.querySelector('#close');
+closeBtnBottom = document.querySelector('#close-button')
```
Comment (developer-anand) [inline components/modal/modal.js]: use ; if you are using it everywhere, just for consistency of code.

File: components/modal/modal.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+openModal.addEventListener('click' , () => {
+
```
Comment (developer-anand) [inline components/modal/modal.js]: remove extra spaces.

**Review by developer-anand — APPROVED** [review]
Nice Work Clean Code and Good naming conventions, for me.
Can be merged.

**Review by arjun-computer-geek — APPROVED** [review]
Looks good.

File: components/modal/modal.js (hunk 3 above)
Comment (Shubhamkr2610) [reply components/modal/modal.js]: okay , I will fix this

**Pull request merged**